docs/create_products.rst

```diff
@@ -0,0 +1,74 @@
+.. _create_products:
+
+Create Jdaviz-readable Products
+===============================
+
+Data products can be loaded in the different Jdaviz configurations using
+various methods, e.g. loading :class:`~specutils.Spectrum1D` objects, loading
+from fits files, or loading a set of fits files from a directory.
+Here, we list a few ways in which data can be packaged to be easily loaded
+into a ``jdaviz`` configuration.
+
+Data in a database
+------------------
+
+If the data are stored in a database, we recommend storing a :class:`~specutils.Spectrum1D` object
+per entry. This would allow the user to query the data and visualize it in
+Jdaviz with few lines of code. This is an example with an imaginary database.
+
+.. code-block:: python
+
+    from database import Client
```

Comment:
This example looks awfully specific. Is this from NOIRLab Sparkle? Maybe better to just link to their doc for this part, and then go from there.

If you must have standalone example, better to use astroquery API that someone else can reproduce without installing NOIRLab specific software.

Reply:
I was hoping it would like generic enough. I'll remove the example.